internal/storage/v2/clickhouse/tracestore/dbmodel/to_dbmodel.go

```diff
@@ -0,0 +1,422 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package dbmodel
+
+import (
+	"encoding/base64"
+
+	"github.com/ClickHouse/ch-go/proto"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+// NestedAttributesGroup There is a one-to-many relationship between a NestedAttributesGroup and a pcommon.Map.
+// In the OTel Collector Contrib implementation, ptrace.SpanEventSlice and ptrace.SpanLinkSlice are stored in a Nested format in the database.
+// Since all arrays in Nested need to have the same length, AttributesGroup cannot be used directly.
+type NestedAttributesGroup struct {
+	AttributesGroups []AttributesGroup
+}
+
+// AttributesGroup There is a one-to-one relationship between an AttributesGroup and a pcommon.Map.
+// In the OTel Collector Contrib implementation, pcommon.Map is stored as a string Map in the database for both key and value,
```

Comment:
asking again - what is the purpose of referencing "OTel Collector Contrib implementation"? I would prefer the comments to the struct to reflect "what is" not "what could've been". If there are differences in how the data is represented here vs. OTEL Contrib then let's write the explanation in the README, in a section "Comparison with OTEL Contrib ClickHouse Exporter"